treeherder/model/management/commands/cycle_data.py

```diff
@@ -22,6 +24,13 @@ def add_arguments(self, parser):
             default=False,
             help='Write debug messages to stdout'
         )
+        parser.add_argument(
+            '--target',
```

Comment:
What do you think about "type" as an alternative to "target"? (I don't mind either way :))